ui/src/dashboards/components/CellEditorOverlay.js

```diff
@@ -63,6 +73,111 @@ class CellEditorOverlay extends Component {
     }
   }
 
+  handleAddThreshold = () => {
+    const {colors} = this.state
+
+    if (colors.length <= MAX_THRESHOLDS) {
+      const randomColor = _.random(0, GAUGE_COLORS.length)
+
+      const maxValue = Number(
+        colors.find(color => color.type === COLOR_TYPE_MAX).value
+      )
+      const minValue = Number(
+        colors.find(color => color.type === COLOR_TYPE_MIN).value
+      )
```

Comment:
Throughout the app we generally use the plus sign for number type coercion, e.g.:
```js
const maxValue = +colors.find(color => color.type === COLOR_TYPE_MAX).value
```

Reply:
Also, be sure you actually need to do that, knowing that JSON preserves all JS types.